pkg/json.go

```diff
@@ -124,6 +124,7 @@ func (r *ScorecardResult) AsJSON2(showDetails bool,
 	}
 
 	encoder := json.NewEncoder(writer)
+	encoder.SetIndent("", "  ")
```

Comment:
There are tools like BigQuery transfers that expect the JSON output to be on a single line. I'm not sure it's our job to prettify the result (breaking change). I fully agree we need more human-friendly results for the badges; but is this the right place for the fix?

Reply:
This is a start. @azeemshaikh38 will this cause breaking charges for BigQuery?

Reply:
I dont think it will for our integration, because we JSONify the results in a different file (under cron/). Some users may be using this output, though.

Reply:
+1. Very likely that some users get broken by this.

Reply:
OK, If that is the case. I will close it 👍

Reply:
So @azeemshaikh38 If I change the BQ will it be an issue?

Reply:
@azeemshaikh38  We discussed this in our bi-weekly. We agreed formatting the JSON output shouldn't be an issue.